GCTC-NTGC/gc-digital-talent
# Bug - 3864 - Add missing skill defs

Resolves #3864 

## Summary

Adds skill definitions where missing as well as handles when a definition does not exist.

## Screenshots

<img width="549" alt="Screen Shot 2022-09-09 at 10 25 22 AM" src="https://user-images.githubusercontent.com/4127998/189373005-69d69cb1-6414-4521-bdec-36c7b22db7aa.png">

<img width="1135" alt="Screen Shot 2022-09-09 at 10 26 30 AM" src="https://user-images.githubusercontent.com/4127998/189373198-0d48e4fc-453d-4d1f-8c9c-e1f9a0f13e19.png">

<img width="942" alt="Screen Shot 2022-09-09 at 10 27 01 AM" src="https://user-images.githubusercontent.com/4127998/189373298-d3be0e5e-2215-4268-9a08-a474c340e161.png">

## Testing

1. Compile intl `npm run intl-compile --workspaces`
2. Build app `npm run production --workspaces`
3. Navigate to `/search`
4. Confirm definitions exist in skill picker
5. Search for "Government"
6. Confirm the "See Definition" button is replaced by "No definition provided"
7. Navigate to a pool advertisement `/browse/pools/{id}/view`
8. Confirm definitions exist in the skill accordions
